test/physics/em/MollerBhabha.test.cc

```diff
@@ -265,12 +265,16 @@ TEST_F(MollerBhabhaInteractorTest, stress_test)
 {
     RandomEngine& rng = this->rng();
 
-    const int           num_samples = 1e5;
+    const int           num_samples = 1e5; // Must be an even number
     std::vector<double> avg_engine_samples;
 
-    for (double inc_e : {1.0, 10.0, 100.0, 1000.0})
+    // Moller's max energy fraction is 0.5, which leads to E_K > 2e-3
+    // Bhabha's max energy fraction is 1.0, which leads to E_K > 1e-3
+    // Since this loop encompasses both Moller and Bhabha, the minimum chosen
+    // energy is > 2e-3.
+    // NOTE: As E_K -> 2e-3, engine_samples -> infinity
+    for (double inc_e : {5e-3, 1.0, 10.0, 100.0, 1000.0})
     {
```

Comment:
The `Interactor` isn't constructed  any differently for these two cases, I was trying to suggest earlier:
```suggestion
for (auto particle : {pdg::electron(), pdg::positron()})
{
    for (double inc_e : {5e-3, 1.0, 10.0, 100.0, 1000.0})
    {
```

And if the rejection loop is never accepts at the lowest energies, shouldn't the lower bound in the `Model` be adjusted to prevent that?

Reply:
That would entail rethinking the implementation, since `min_energy_fraction_ = min_valid_energy / inc_energy`. By moving the lower bound (`min_valid_energy`) we just end up creating a new limit at which our sampling loop is going to work.

Reply:
The `min_valid_energy` is different from the `Applicability.lower`. We could keep the single lower energy (1e-3) for the model data,  but we should double the applicability limit for electrons so it matches the zero value for the cross sections and won't go into an infinite rejection loop.